### DISCUSSION TITLE
Administrador de complementos

### DISCUSSION THREAD

**Discussion by riopar10** [FreeCAD/FreeCAD#10184] [Code contribution discussions]
No he encontrado la solución a este problema. - El proceso en memoria anterior fue interrumpido, reiniciando...Administrador de complementos: Respuesta 0 inesperada del servidor
Error al conectar a GitHub. Compruebe su conexión y configuración de proxy. 
Podeis ayudarme para no tener este problema en freecad, gracias

OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.21.0.33668 +7 (Git)
Build type: Release
Branch: (HEAD detached at 0.21)
Hash: 41b058e2087de60dc8fef3d3e68c7d0129e13abf
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: Spanish/Spain (es_ES) [ OS: Spanish/Spain (es_ES) ]

**Comment by luzpaz** [comment]
This looks like an issue with your internet connection or an issue between you and https://github.com

Note: Better to open a forum thread to discuss with the community https://forum.freecad.org/viewforum.php?f=14
